pyboy/plugins/debug.py

```diff
@@ -4,22 +4,26 @@
 #
 
 import ctypes
+import os
 from array import array
 
 import sdl2
+import sdl2.sdlttf as sdl2_ttf
 from pyboy.botsupport import constants, tilemap
 from pyboy.botsupport.sprite import Sprite
 from pyboy.plugins.base_plugin import PyBoyWindowPlugin
 from pyboy.plugins.window_sdl2 import sdl2_event_pump
 from pyboy.utils import WindowEvent
 from pyboy.logger import logger
 
-
 try:
-    from cython import compiled
+    from cython import compiled, NULL
+    import cython
     cythonmode = compiled
 except ImportError:
     cythonmode = False
+    exec("NULL = None", globals(), locals())
```

Comment:
This is a really clever trick, and I love it. We might be able to work that into some of the other SDL calls.